pkg/common/diskutil/file_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !windows
```

Comment:
Is file_windows_test.go a duplicate of this file? If so, would it be better to remove the build constraints from this file?

Reply:
I see now that there are mode differences... hmm, i'm ok either way but one alternative is to detect the os at runtime and change the expected mode accordingly.